### PR TITLE
PORTALS-386: Authorize createInvalidations

### PR DESCRIPTION
**Pull request by xschildw**
Per discussion with Khai, best to have the job take care of it.

### CODE REVIEW

File: templates/accounts.yaml [hunk 1]
```diff
@@ -248,6 +248,12 @@ Resources:
           - arn:aws:s3:::static.nf.synapse.org/*
           - arn:aws:s3:::prod.csbc-pson.synapse.org/*
           - arn:aws:s3:::staging.csbc-pson.synapse.org/*
+        - Effect: Allow
+          Action:
+          - cloudfront:CreateInvalidation
+          Resource:
+            - arn:aws:cloudfront::797640923903:distribution/E284T8G8XOYXA7
```
Comment (zaro0508) [inline templates/accounts.yaml]: Would be better if these are referenced from an export so you can do an `!ImportValue` here

Comment (xschildw) [reply templates/accounts.yaml]: Thought about it but the only thing I see is DomainName. From the doc it does not look like !GetAtt resource.Arn is always available, same for Id if I wanted to build the arn in the template. Is there another way to do it?

Comment (xschildw) [reply templates/accounts.yaml]: Thinking about it, should accounts.yaml have dependency on explicit resources (i.e. should I just not put *)?

Comment (zaro0508) [reply templates/accounts.yaml]: You're right the `AWS::CloudFront::Distribution` doesn't support a ref to the ARN however you can build the ARN like so..
```
- Join
  - ''
  - - 'arn:aws:cloudfront::'
  - - !Ref 'AWS:AccountId'
  - - ':distribution/'
  - - !ImportValue <region>-<stack_name>-CloudfrontId
```

That assumes that you have defined an export value somewhere that's `<region>-<stack_name>-CloudfrontId`

**Comment by xschildw** [comment]
Exported cfront distribution id (separate PR in aws-infra) and using here.

**Pull request closed without merging**